src/BlazorUI/Bit.BlazorUI/Styles/general.scss

```
@@ -1,4 +1,4 @@
-﻿@import "functions.scss";
+@import "functions.scss";
```

Stylelint (17.5.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)